src/ledgers/five-bells-ledger.js

```diff
@@ -172,6 +179,96 @@ class FiveBellsLedger extends EventEmitter {
     return fulfillmentRes.body
   }
 
+  * _handleNotification (fiveBellsTransfer, relatedResources) {
+    if (fiveBellsTransfer.ledger !== this.id) {
+      throw new Error('Transfer was received by the wrong plugin (plugin: ' +
+        this.id + ', transfer: ' + fiveBellsTransfer.ledger + ')')
+    }
+
+    let handled = false
+    for (let credit of fiveBellsTransfer.credits) {
+      if (credit.account === this.credentials.account_uri) {
+        handled = true
+
+        const transfer = lodash.omitBy({
+          id: fiveBellsTransfer.id.substring(fiveBellsTransfer.id.length - 36),
+          direction: 'incoming',
+          ledger: this.id,
+          // TODO: What if there are multiple debits?
+          account: fiveBellsTransfer.debits[0].account,
+          amount: credit.amount,
+          data: credit.memo,
+          executionCondition: fiveBellsTransfer.execution_condition,
+          cancellationCondition: fiveBellsTransfer.cancellation_condition,
+          expiresAt: fiveBellsTransfer.expires_at,
+          cases: fiveBellsTransfer.additional_info && fiveBellsTransfer.additional_info.cases
+            ? fiveBellsTransfer.additional_info.cases
+            : undefined
+        }, lodash.isUndefined)
+
+        if (fiveBellsTransfer.state === 'prepared' ||
+            (fiveBellsTransfer.state === 'executed' && !transfer.executionCondition)) {
+          this.validateTransfer(credit.memo.destination_transfer)
+          yield this.emitAsync('incoming', transfer)
+        }
+
+        if (fiveBellsTransfer.state === 'executed' && relatedResources &&
+            relatedResources.execution_condition_fulfillment) {
+          yield this.emitAsync('fulfill_execution_condition', transfer,
+            relatedResources.execution_condition_fulfillment)
+        }
+
+        if (fiveBellsTransfer.state === 'rejected' && relatedResources &&
+            relatedResources.cancellation_condition_fulfillment) {
+          yield this.emitAsync('fulfill_cancellation_condition', transfer,
+            relatedResources.cancellation_condition_fulfillment)
+        }
+      }
+    }
+
+    for (let debit of fiveBellsTransfer.debits) {
+      if (debit.account === this.credentials.account_uri) {
+        handled = true
+
+        // This connector only launches transfers with one credit, so there
+        // should never be more than one credit.
```

Comment:
I should mention: Even if we are processing any such transfers - there isn't really anything bad that can happen since all we do in response to outgoing transfers is fulfill incoming ones. So the gun and the foot are pretty far apart here.